airflow/providers/ssh/hooks/ssh.py

```diff
@@ -471,11 +471,11 @@ def exec_ssh_client_command(
         command: str,
         get_pty: bool,
         environment: dict | None,
-        timeout: int | None = None,
+        timeout: int | ArgNotSet | None = None,
```

Comment:
I feel the default be changed to `NOTSET`. And the `None` case below should be dropped entirely…?

Reply:
updated - much better imho. ty